### ISSUE TITLE
Fix Xubmit NULL ID Error for Newton P5 Repo

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#60]
@randalldfloyd is looking into this error in hopes that he can fix it. After another round of troubleshooting on Friday, 2/21/2020, the problem is definitely with Xubmit or at least how that  repo was created/configured. This next to get fixed before we can wrap up issue #55.

**Comment by randalldfloyd** [comment]
The 'field.xsl' file was missing the tei namespace declaration, so the transformation was producing an empty document. That breaks because Xubmit uses the result to derive fields for all of its internal tracking.  This file is stored server-side per collection in the main Xubmit directory, in this case:

/opt/xubmit/repositories/newtonchym/xslt/field.xsl

Also, notice that the fields that aren't derived from parameters (i.e. xfile, xuser, xlog, etc.) still come up null, which is because of xpath problems in the same XSL. The next step is for someone on the content expert/metadata side to get that transform working properly outside of Xubmit and then update it on the server.

I will update issue #60 and create a new issue for the missing fields problem.

**Comment by randalldfloyd** [comment]
Created #74 to fix missing fields found while solving Null ID error.
